tools/hack/create-cluster.sh

```diff
@@ -6,12 +6,16 @@ set -euo pipefail
 CLUSTER_NAME=${CLUSTER_NAME:-"envoy-gateway"}
 METALLB_VERSION=${METALLB_VERSION:-"v0.12.1"}
 KIND_NODE_TAG=${KIND_NODE_TAG:-"v1.24.0"}
+readonly HERE=$(cd "$(dirname "$0")" && pwd)
+readonly REPO=$(cd "${HERE}/../.." && pwd)
+readonly CONFIG_FILE="${REPO}/tools/hack/kind-expose-ports.yaml"
```

Comment:
Ugh, I hate that these types of hacks are necessary.  We solved this long ago with `realpath`, but macOS is holding the world back...

But wait, we have `git`!
```bash
REPO=$(git rev-parse --show-toplevel)
```